Formula/zlog.rb

```diff
@@ -0,0 +1,48 @@
+class Zlog < Formula
+  desc "High-performance C logging library"
+  homepage "https://github.com/HardySimpson/zlog"
+  url "https://github.com/HardySimpson/zlog/archive/1.2.12.tar.gz"
+  sha256 "9c6014a3f74d136c70255539beba11f30e1d3617d07ce7ea917b35f3e52bac20"
+
+  def install
+    system "make"
```

Comment:
Does this need a separate `make` step or will it compile & install fine with just:
```ruby
system "make", "PREFIX=#{prefix}", "install"
```
?

Reply:
https://github.com/HardySimpson/zlog/blob/master/README.md#L23

Reply:
It will compile with `system "make"` & install with `system "make", "PREFIX=#{prefix}", "install"`